### PR TITLE
Minor updates to geospatial functions/handling

### PR DESCRIPTION
**Pull request by ConnectedSystems**
Some small maintenance changes.

### CODE REVIEW

File: src/io/result_io.jl [hunk 1]
```diff
@@ -369,13 +369,13 @@ function setup_result_store!(domain::Domain, scen_spec::DataFrame)::Tuple
     mkdir(joinpath(log_location, SPATIAL_DATA))
     geo_fn = joinpath(log_location, SPATIAL_DATA, basename(attrs[:name]) * ".gpkg")
     try
-        GDF.write(geo_fn, domain.site_data; driver="geojson")
+        GDF.write(geo_fn, domain.site_data; driver="geopackage")
```
Comment (DanTanAtAims) [inline src/io/result_io.jl:372]:
```suggestion
        GDF.write(geo_fn, domain.site_data; driver="GPKG")
```
Is `GPKG` the intended driver?

using `geopackage` causes a GDAL error.
```
ERROR: GDALError (CE_Failure, code 10):
        Pointer 'hDriver' is NULL in 'GDALCreate'.
```

File: src/io/result_io.jl [hunk 2]
```diff
@@ -369,13 +369,13 @@ function setup_result_store!(domain::Domain, scen_spec::DataFrame)::Tuple
...
     catch err
         if !isa(err, ArgumentError)
             rethrow(err)
         end
 
-        GDF.write(geo_fn, domain.site_data; geom_columns=(:geom,), driver="geojson")
+        GDF.write(geo_fn, domain.site_data; geom_columns=(:geom,), driver="geopackage")
```
Comment (DanTanAtAims) [inline src/io/result_io.jl:378]: same as above.

File: src/io/result_io.jl (hunk 1 above)
Comment (ConnectedSystems) [reply src/io/result_io.jl:372]: Ah, I thought I had changed that!

File: src/io/result_io.jl [hunk 3]
```diff
@@ -369,13 +369,14 @@ function setup_result_store!(domain::Domain, scen_spec::DataFrame)::Tuple
...
     try
-        GDF.write(geo_fn, domain.site_data; driver="geopackage")
+        GDF.write(geo_fn, domain.site_data; driver="GPKG")
     catch err
         if !isa(err, ArgumentError)
             rethrow(err)
         end
 
-        GDF.write(geo_fn, domain.site_data; geom_columns=(:geom,), driver="geopackage")
+        col = _get_geom_col(domain.site_data)
```
Comment (DanTanAtAims) [inline src/io/result_io.jl:378]: Should we move this into the main package?
```
ERROR: UndefVarError: `_get_geom_col` not defined
```

Comment (ConnectedSystems) [reply src/io/result_io.jl:378]: Ugh, my bad again. Must've had a global copy loaded. I'll sort it out

**Comment by ConnectedSystems** [comment]
Found some more bugs when trying to resolve the previous issue.

Tests will still fail but for unrelated reasons (more PRs incoming to try to resolve those, but some will need input from @Zapiano).

**Comment by ConnectedSystems** [comment]
@Rosejoycrocker This PR will now resolve #800 , at least for ADRIA tests. 
The synthetic data generation will need to create geopackages with a valid CRS.

**Comment by Rosejoycrocker** [comment]
> @Rosejoycrocker This PR will now resolve #800 , at least for ADRIA tests. The synthetic data generation will need to create geopackages with a valid CRS.

Thanks @ConnectedSystems, noted. The synthetic GeoDataframe itself has a defined CRS so I think I just need to specify this when it's saved to a geopackage

**Comment by ConnectedSystems** [comment]
For the record, the geopackages in this PR were updated manually by attaching CRS WGS84 (EPSG:4326) in QGIS.

**Review by DanTanAtAims — APPROVED** [review]
Looks good to me.

**Pull request merged**